tests/migrations/test_commands.py

```diff
@@ -451,6 +451,18 @@ def test_showmigrations_plan_squashed(self):
             out.getvalue().lower()
         )
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed_twice"})
+    def test_showmigrations_plan_squashed_twice(self):
+        """
+        Tests --plan output of showmigrations command with squashed migrations.
+        """
+        out = io.StringIO()
+        call_command("showmigrations", format='plan', stdout=out)
```

Comment:
Doesn't happen all the time, but this call _sometimes_ raises an exception when ran locally: `Unable to find replacement node ('migrations', '0001_squashed_0002'). It was either never added to the migration graph, or has been removed.`

```
======================================================================
ERROR [0.072s]: test_migrate_record_squashed_twice (migrations.test_commands.MigrateTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/local/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/tests/django/django/test/utils.py", line 370, in inner
    return func(*args, **kwargs)
  File "/tests/django/tests/migrations/test_commands.py", line 793, in test_migrate_record_squashed_twice
    call_command("migrate", "migrations", verbosity=0)
  File "/tests/django/django/core/management/__init__.py", line 168, in call_command
    return command.execute(*args, **defaults)
  File "/tests/django/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/tests/django/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/tests/django/django/core/management/commands/migrate.py", line 86, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/tests/django/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/tests/django/django/db/migrations/loader.py", line 49, in __init__
    self.build_graph()
  File "/tests/django/django/db/migrations/loader.py", line 241, in build_graph
    self.graph.remove_replaced_nodes(key, migration.replaces)
  File "/tests/django/django/db/migrations/graph.py", line 137, in remove_replaced_nodes
    ) from err
django.db.migrations.exceptions.NodeNotFoundError: Unable to find replacement node ('migrations', '0001_squashed_0002'). It was either never added to the migration graph, or has been removed.
```